Tests/MongoSwiftSyncTests/SDAMTests.swift

```diff
@@ -46,23 +46,27 @@ final class SDAMTests: MongoSwiftTestCase {
             return
         }
 
-        expect(receivedEvents.count).to(equal(4))
+        expect(receivedEvents.count).to(equal(5))
```

Comment:
so, the CDRIVER ticket mentioned above only actually resolved the duplicate server changed event issue. The topology changed one still existed. however, according to [DRIVERS-944](https://jira.mongodb.org/browse/DRIVERS-944), publishing the event is consistent with the SDAM monitoring tests.

I'm kind of 50/50 on whether to make the change, because as Kevin points out in the comment, there is no visible change to the user from the topology description. But to not publish it would be inconsistent with the spec and what other drivers seem to do if they pass the test.

Reply:
Yeah I kind of agree that we probably shouldn't emit events in this case, but also think for simplicity and conformity we should just do what the tests require. If that ticket ever gets resolved in the other direction we could just stop emitting the event--I don't think it would constitute a breaking behavioral change.